core/src/data/value/mod.rs

```diff
@@ -555,6 +555,41 @@ impl Value {
         }
     }
 
+    pub fn bitwise_shift_left(&self, rhs: &Value) -> Result<Value> {
+        use Value::*;
+
+        let rhs = u32::try_from(rhs)?;
```

Comment:
Oh, I missed one.
in the literal
the result of `SELECT 10 << NULL` is `NULL`.
but for the value, it will return error.
for the consistency, it would be better to add logic to return `NULL` when `rhs` is null